### DISCUSSION TITLE
2005/06/14/%E4%BB%A5%E5%BE%80%E8%AE%A1%E6%95%B0%E5%99%A8%E4%B8%8B%E7%9A%84%E7%95%99%E8%A8%80-4

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#37] [Announcements]
# 2005/06/14/%E4%BB%A5%E5%BE%80%E8%AE%A1%E6%95%B0%E5%99%A8%E4%B8%8B%E7%9A%84%E7%95%99%E8%A8%80-4

https://blog.qizhen.xyz/2005/06/14/%E4%BB%A5%E5%BE%80%E8%AE%A1%E6%95%B0%E5%99%A8%E4%B8%8B%E7%9A%84%E7%95%99%E8%A8%80-4

**Comment by ruanqizhen** [comment]
米
2005年06月20日 13:52 编辑
gxgx～这里人气真足啊！

回复
Unknown
2005年06月19日 19:10 编辑
欢迎来我的Space玩玩

回复
-----Angie-----
2005年06月16日 12:10 编辑
飘过飘过…戴眼镜不好哦,眼睛会凸出滴,象金鱼眼,嘿嘿:)

回复
哈米の哟哟
2005年06月15日 00:27 编辑
恩~不错

回复
PP
2005年06月14日 23:49 编辑
以后也来偷东西,嘻嘻:)加油加油,你的space很不错!!!

回复
珍珠变鱼眼
2005年06月14日 15:49 编辑
这么多次了,厉害啊!~加油~你做的共享的确不错!

回复
